src/index.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+module.exports = {
+  Relay: require('./circuit/relay'),
+  Dialer: require('./dialer'),
+  multicodec: require('./multicodec'),
+  tag: 'Circuit'
```

Comment:
Since circuit pretends to be a transport, its used by swarm/libp2p to when loading all transports.